mage_ai/server/websocket_server.py

```diff
@@ -321,11 +321,15 @@ def filter_out_sensitive_data(message):
             else WebSocketServer.running_executions_mapping.get(msg_id, dict())
         block_type = msg_id_value.get('block_type')
         block_uuid = msg_id_value.get('block_uuid')
+        replicated_block = msg_id_value.get('replicated_block')
         pipeline_uuid = msg_id_value.get('pipeline_uuid')
 
         error = message.get('error')
         if error:
-            message['data'] = cls.format_error(error, block_uuid=block_uuid)
+            message['data'] = cls.format_error(
+                error,
+                block_uuid=replicated_block if replicated_block else block_uuid,
```

Comment:
How is this bock_uuid used?

Reply:
the `block_uuid` here is used to search through the logs. a replica block just directly runs the code of the `replicated_block`, so that's why i'm passing in the `replicated_block`